k8s/upgrades/0.5.x-0.6.0/controller.patch.tpl.yml

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Can we write yaml instead of json. Advantage being we can write comments about the patch. Additional advantages of yml over json also applies i.e. no housekeeping stuff like `{` `}` & `"`

I have assumed that kubectl treats yaml & json with equal ease even for patch operation.

The file name is .yml though.
Ideally it should be `.json.tpl` that gets transformed to `.json`